### ISSUE TITLE
Allow comments in module JSON

### ISSUE THREAD

**Issue by serin-delaunay** [Wangscape/Wangscape#195]
To do this, add an optional string field named "comment" to the module schemas ~and codecs~, so that the following is valid:
```json
    "corner_combiner_base":{
      "type":"CornerCombinerBase",
      "Power":2,
      "comment": "0.5 along diagonals, 1 or -1 along axes, gradient in between. Power affects the sharpness of the gradient."
    },
```
This will allow module groups to be documented in-line, making them easier to understand or alter.

**Comment by V1shvesh** [comment]
anyone assigned? I wish to make my first contribution and would like you to help me help you :stuck_out_tongue:

**Comment by hryniuk** [comment]
@V1shvesh no, you can start working on it. Looking forward to your PR :).

**Comment by hryniuk** [comment]
@serin-delaunay would you like to add something? Is it enough to change just one codec, i.e. `NoiseSourcesCodec` you've mentioned yesterday?

@V1shvesh if you have some questions about Wangscape, but not connected to this issue, you can ask them in our Gitter room: https://gitter.im/Wangscape/Lobby.

**Comment by serin-delaunay** [comment]
Hi @V1shvesh, welcome to the project! I should add some clarification:

The `"comment"` field in a module would be processed by the [module group schema](https://github.com/Wangscape/Wangscape/blob/master/doc/schemas/module_group_schema.json), and by some spotify-json codecs (`NoiseSourcesCodec` and `*WrapperCodec`). The spotify-json codecs don't cause errors when an object has additional properties, so only the JSON schema needs to be changed.

**Comment by V1shvesh** [comment]
Could you guys let me know what would be a valid value for`"title"` field in `"comment"`?

**Comment by serin-delaunay** [comment]
`"Comment schema"` or `"Module comment schema"` would be appropriate.

**Comment by serin-delaunay** [comment]
Fixed in #205.
